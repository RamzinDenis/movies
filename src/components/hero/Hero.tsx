import React from "react";
import styles from "./hero.module.sass";

const Hero: React.FC = ({ children }) => {
	return (
		<main className={styles.hero}>
			<div className={styles.container}>{children}</div>
		</main>
	);
};

export default Hero;
