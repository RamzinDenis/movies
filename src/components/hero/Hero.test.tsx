import React from "react";
import { shallow } from "enzyme";
import toJson from "enzyme-to-json";
import Hero from "./";
describe("<Hero />", () => {
	it("should render Hero component correctly", () => {
		const component = shallow(<Hero />);
		expect(toJson(component)).toMatchSnapshot();
	});
});
