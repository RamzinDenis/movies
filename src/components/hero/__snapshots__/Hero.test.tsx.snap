// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Hero /> should render Hero component correctly 1`] = `
<main
  className="hero"
>
  <div
    className="container"
  />
</main>
`;
